tests/test_cases/test_cocotb/test_concurrency_primitives.py

```diff
@@ -99,6 +99,34 @@ async def coro():
     assert ran
 
 
+# currently fork causes the test to fail, but this may change
+@cocotb.test(expect_error=ValueError)
+async def test_exceptions_forked_late(dut):
+    """Test exception propogation via fork, but await *after* forked coro has raised"""
+
+    async def raise_soon():
+        raise ValueError('It is soon now')
+
+    async def await_later():
+        future = cocotb.fork(raise_soon())
+        await Timer(1)
```

Comment:
Does this need to be 1? What happens if it's 0 or 2?

Reply:
`Timer(0)` can sometime cause issues, it just has to be shorter than the `await` in the main coroutine below. I has though about turning this into an event.

Reply:
My question was in particular about whether this has to match the `Timer(1)` within the scheduler.

Reply:
No, it's arbitrary. I'll make it an event, I think it will be more clear that way.

Reply:
Now that my bearings are straight; no, it does not have to be 1. I tried with 2 and it fails in the same way.